kong/plugins/apifortress/schema.lua

```diff
@@ -0,0 +1,23 @@
+-- The plugin schema
+
+local function validateKeys(value)
+  return #value > 0
+end
+
+local function validateThreshold(value)
+  return value > 0
+end
+
+local function validateProjectId(value)
+  return value > 0
```

Comment:
Because they might have different behaviors in the future, so I decided to keep the semantic clear